### DISCUSSION TITLE
Can't get CHIA to OKX wallet. (Sending delayed by 15 min.).

### DISCUSSION THREAD

**Discussion by PovAndy** [Chia-Network/chia-blockchain#18192] [Support]
I made a transfer of funds from my Chia wallet xch1gd5gcjrksyxhz7qxkmcsgtedy8djejpeskqk8ka7z0aadvazmqksesc45n transfer in the amount of 3 XCH with a delay of 15 minutes.
The transfer was made to my address in OKX - xch1pkqau9kf90fhx3d54qx3l7h6ffzv40xewjw4rsyw9p5knqgeh0squze2qe . (Screenshot of the 3 XCH translation attached).
I made the transfer and after 10 minutes it stopped and displayed a message as in screenshot 2 (message "Pending claw back transactions").
What should I do, how can I get my funds to my OKX wallet?

I found a transaction by block number 5,503,188, the sending time matches, but the wallet numbers (sender and receiver) do not match!

On OKX wallet there is no button to receive a pending transfer, so I can't receive coins to OKX wallet.

What should I do, how can I send coins to my OKX wallet or get them back?

OKX tech support says that the wallet numbers don't match and they can't credit my coins to my OKX wallet!

![2024-06-17_14-10-26](https://github.com/Chia-Network/chia-blockchain/assets/45090797/2296069d-7663-4b3d-b56c-d9e1377fca3f)
![2024-06-17_14-13-02](https://github.com/Chia-Network/chia-blockchain/assets/45090797/ec3f8cdc-3551-48de-9109-942d8e84833c)
![2024-06-17_14-19-50](https://github.com/Chia-Network/chia-blockchain/assets/45090797/0313e658-e40a-432c-be0f-a63d383af734)
![2024-06-17_14-20-29](https://github.com/Chia-Network/chia-blockchain/assets/45090797/1b2018df-4d40-42db-9fdd-a83dc1f3ed75)
![2024-06-17_14-23-50](https://github.com/Chia-Network/chia-blockchain/assets/45090797/df3625ed-9b62-4c17-b1e8-4b523e1ab4dc)

**Comment by PovAndy** [comment]
After deferred transfer for 3 XCH I made 2 simple transfers (2 and 0.87 XCH) to the same OKX wallet, all coins arrived normally.

![2024-06-17_11-42-33](https://github.com/Chia-Network/chia-blockchain/assets/45090797/970cf07d-0d78-4a7d-bad6-b33ddf0ab8ce)
![2024-06-17_11-42-58](https://github.com/Chia-Network/chia-blockchain/assets/45090797/937a522c-59a9-46d0-9395-b74aecc86494)

**Reply by PovAndy** [reply]
This is the way it worked for me:
> The first thing to try is resyncing your wallet.  Open Chia and select Settings -> Advanced -> Resync.  This will close the Chia app after a few seconds.  Wait a moment and reopen Chia.  Once it is open, wait for the WALLET icon in the top right corner of the screen to turn green.  This will indicate that the wallet is synced.  Then check your transaction list to see if the clawback is still stuck.  If it is, there are few more tricks we can try.  Let us know how it goes or if you run into trouble.

**Comment by BrandtH22** [comment]
Hey @PovAndy , thank you for jumping into the discord and working with the support team there. I am glad to hear the issue is resolved and will be closing this discussion. 
If you have other questions or run into any issues feel free to reopen this ticket if they are related, open a new ticket, or for the most timely and thorough support jump back into our discord server (https://discord.gg/chia)
